site/gatsby-site/src/templates/report.js

```diff
@@ -84,7 +85,35 @@ function ReportPage(props) {
       />
 
       <Container className="mt-4">
-        <ReportCard item={report} alwaysExpanded={true} actions={actions} />
+        {incidents.nodes.length > 0 && (
+          <Card className={'shadow-card'} data-cy="classifications-editor">
+            <Card.Header className="items-center justify-between">
+              <h4>
+                <Trans>Associated Incidents</Trans>
+              </h4>
+            </Card.Header>
+
+            <Card.Body>
+              {incidents.nodes.map((incident) => (
+                <Link to={`/cite/${incident.incident_id}`} key={incident.incident_id}>
+                  <h4 className="mb-2 text-md font-bold tracking-tight text-gray-900 dark:text-white">
+                    <span className="text-sm">
+                      <Trans>Incident {{ id: incident.incident_id }}</Trans>
+                    </span>
+                    <span className="ml-2 bg-red-100 text-red-800 text-xs font-semibold mr-2 px-2.5 py-0.5 rounded dark:bg-green-200 dark:text-green-900">
+                      <Trans ns="entities" count={incident.reports.length}>
```

Comment:
`ns="reports"`? or am I about to learn something?

Reply:
(but this info tag was a nice touch, user was delighted)

Reply:
Those namespaces refer to the files in this folder: https://github.com/responsible-ai-collaborative/aiid/tree/master/site/gatsby-site/i18n/locales

Only the `translation.json` file is loaded by default, and all others are requested on demand. If a text string is used in multiple places, we place it there. I made some rearrangements here to reflect this.